man/systemd.unit.xml

```diff
@@ -495,14 +493,24 @@
       <varlistentry>
         <term><varname>BindsTo=</varname></term>
 
-        <listitem><para>Configures requirement dependencies, very
-        similar in style to <varname>Requires=</varname>, however in
-        addition to this behavior, it also declares that this unit is
-        stopped when any of the units listed suddenly disappears.
-        Units can suddenly, unexpectedly disappear if a service
-        terminates on its own choice, a device is unplugged or a mount
-        point unmounted without involvement of
-        systemd.</para></listitem>
+        <listitem><para>Configures requirement dependencies, very similar in style to
+        <varname>Requires=</varname>. However, this dependency type is stronger: in addition to the effect of
+        <varname>Requires=</varname> it declares that if the unit bound to is stopped, this unit will be stopped
+        too. This means a unit bound to another unit that suddenly enters inactive state will be promptly stopped too.
+        Units can suddenly, unexpectedly enter inactive state for different reasons: the main process of a service unit
+        might terminate on its own choice, the backing device of a device unit might be unplugged or the mount point of
+        a mount unit might be unmounted without involvement of the system and service manager.</para>
+
+        <para>When used in conjunction with <varname>After=</varname> on the same unit the behaviour of
+        <varname>BindsTo=</varname> is even stronger. In this case, the unit bound to strictly has to be in activated
+        state for this unit to also be in activated state. This not only means a unit bound to another unit that
+        suddenly enters inactive state, but also one that is bound to another unit that gets skipped due to a failed
+        condition check (such as <varname>ConditionPathExists=</varname>,
+        <varname>ConditionPathIsSymbolicLink=</varname>, … — see below) will be promptly stopped, should it be
+        running.</para>
+
+        <para>Note that <varname>BindsTo=</varname> does not imply an ordering dependency, but in many cases is best
+        combined with <varname>After=</varname>, see above.</para></listitem>
```

Comment:
It'd merge the last two paragraphs. That last sentence could just be "Hence, in many cases it is best to combine BindsTo= with After=.".